.github/workflows/build.yml

```
@@ -96,7 +96,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        cmd: ['ci-webdriver-standard', 'ci-e2e-integration', 'ci-webdriver-default-mobile']
+        cmd: ['ci-webdriver-standard', 'ci-e2e-integration', 'ci-webdriver-default-mobile', 'ci-webdriver-standard-minimum-chrome']
```

I think this will cause a lot of duplication once we have all of the other suites also running Chrome 90. Ideally I think we'd have a matrix for the browser versions ([docs](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)), eg:

```suggestion
        cmd: ['ci-webdriver-standard', 'ci-e2e-integration', 'ci-webdriver-default-mobile'],
        chrome-version: ['90', 'latest']
```

Then pass the chrome version as an environment variable and use that everywhere. This makes it easy to roll out the other test suites without needing a whole new npm run command.

This causes a problem in the short term because it'll automatically run all 3 commands for both chrome versions (6 builds total), and you only have one working for now, but I think you can add something in the suite to skip for unsupported chrome versions until you get to the PR to fix it.

---

Yes! I agree. At first, I wanted to implement this using the same wdio.conf.js and add an extra capability for Chrome 90. But I couldn't make it work because we already use Devtools as a service for Chrome latest, and when adding Chromedriver to the list of services, the first failed. So I have to have a separate conf for Chromedriver.
I will use an env variable to choose between wdio.conf.js files. Does this make sense?

---

I think you should be able to include a switch in the wdio.conf.js to include devtools _or_ chromedriver. For prior art look at how we're switching "headless" based on the environment variable for "DEBUG": https://github.com/medic/cht-core/blob/master/tests/e2e/wdio.conf.js#L86

Would this work here too?